### PR TITLE
Fix for download videos

### PR DESCRIPTION
**Pull request by TainakaDrums**

### CODE REVIEW

File: tumblr-photo-video-ripper.py [hunk 1]
```diff
@@ -107,6 +107,7 @@ def _download(self, medium_type, medium_url, target_folder):
                                         medium_name])
 
             medium_name += ".mp4"
+            medium_url = 'https://vt.tumblr.com/' + medium_name
```
Comment (dixudx) [inline tumblr-photo-video-ripper.py:110]: How comes this url?

Comment (TainakaDrums) [reply tumblr-photo-video-ripper.py:110]: From web version

Comment (dixudx) [reply tumblr-photo-video-ripper.py:110]: @TainakaDrums I'm not sure for this change. Can anybody help verify this?

**Comment by MarsDDD** [comment]
It works! Thanks a lot!
Can you explain it to me a bit though?

File: tumblr-photo-video-ripper.py (hunk 1 above)
Comment (WeibingChen17) [reply tumblr-photo-video-ripper.py:110]: I can confirm that the previous medium_url is forwarded to this new medium_url in browsers

**Pull request merged**